GoogleDataTransport/ProtoSupport/Protos/client_metrics.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package gdt_client_metrics;
+
+// Top level metrics for all client analytics metrics.
+// These metrics should be sent as a part of every request that is uploaded to
+// FireLog server. In more detail, an additional LogRequest should be added to
+// the BatchedLogRequest, where the LogSource of the LogRequest should be
+// GDT_CLIENT_METRICS and the LogRequest should have a single LogEvent whose
+// payload is a GoogleDataTransportClientMetrics message.
+//
+// Original proto: google3/firebase/firelog/proto/gdt_client_analytics.proto
+// See go/firelog-client-analytics for more details.
+message ClientMetrics {
+  // The window of time over which the metrics are evaluated.
+  TimeWindow window = 1;
+
+  repeated LogSourceMetrics log_source_metrics = 2;
+
+  GlobalMetrics global_metrics = 3;
+
+  // The bundle ID on Apple platforms (e.g., iOS) or the package name on Android
+  string app_namespace = 4;
```

Comment:
Based on DD I'm referencing, should this be `app_bundle`?

Reply:
I think DD is slightly outdated. Currently the source of truth is the proto at google3/firebase/firelog/proto/gdt_client_analytics.proto which is used by the backend and copied to Apple and Android SDK.